### DISCUSSION TITLE
请教一下超过5秒的回复怎么处理

### DISCUSSION THREAD

**Discussion by wizzhi** [offu/WeRoBot#696] [Q&A]
按照文档， 响应时间超过5秒应该先返回SuccessReply，但是这样的话，在werobot的框架下，后续怎么返回消息呢？因为handler已经返回过SuccessReply了。

**Comment by whtsky** [comment]
可以开一个新的 Thread ，然后使用客服接口发送消息（注意需要权限）
https://werobot.readthedocs.io/zh_CN/latest/client.html#werobot.client.Client.send_text_message

**Reply by wizzhi** [reply]
谢谢指点，不过可能很多人没有权限啊（我也没有）。我感觉应该可以在handler里面允许调用一次success reply的方法，但是handler并不返回，等执行完了之后返回的时候后台另外reply 一条消息。这样的话对于调用者是最简单的。

**Reply by whtsky** [reply]
用客服接口其实就是“后台另外reply一条消息”，据我所知微信服务器不支持客服接口以外的“后台reply消息”的方式了。

**Reply by wizzhi** [reply]
好滴，我再翻翻看，感觉如果回复success都不让再跟一条的话，有点不太合理啊～

**Reply by mengmajun** [reply]
请问 你解决了吗

**Reply by wizzhi** [reply]
没有解决啊，希望有能力的大侠提个PR吧。。。

**Reply by 1606701** [reply]
同问，顶

**Reply by gaoheran** [reply]
用模板消息可以不受限制的发送
